### PR TITLE
Add Geetika's banner

### PR DESCRIPTION
**Pull request by tkkuehn**
Geetika made this nice banner, so this includes it as the page header.

Screenshot_2019-11-15 AFIDs Validator

### CODE REVIEW

**Comment by kaitj** [comment]
Does afids have a logo? Maybe from the afids git organization (https://github.com/afids)

File: templates/view.html [hunk 1]
```diff
@@ -12,7 +12,8 @@
 
   <!-- Input and Results are typeset as a two-column table -->
   <header class="container">
-    <h1>AFIDs Validator</h1>
+      <img src="{{ url_for('static', filename='banner.png') }}"
```
Comment (Park-Patrick) [inline templates/view.html]: any reason for the {{url_for}} syntax?
just
`AFIDs Validator Banner`
works for me

Comment (tkkuehn) [reply templates/view.html]: Just continuity with lines 6 & 7 - I don't really remember what it does lol...

Comment (Park-Patrick) [reply templates/view.html]: Also i think we can make the same change to `view_analytics.html` for consistency

Comment (tkkuehn) [reply templates/view.html]: I think I just copied this: https://flask.palletsprojects.com/en/1.1.x/quickstart/#static-files

Comment (tkkuehn) [reply templates/view.html]: Ok I'll add that

**Comment by Park-Patrick** [comment]
> Does afids have a logo? Maybe from the afids git organization (https://github.com/afids)


maybe the green bug? or something simple like this could work too.

image

File: templates/view.html (hunk 1 above)
Comment (Park-Patrick) [reply templates/view.html]: thanks, and have we decided on a background color?

**Comment by kaitj** [comment]
> > Does afids have a logo? Maybe from the afids git organization (https://github.com/afids)
> 
> maybe the green bug? or something simple like this could work too.
> 
> image

I think Geetika added the green bug - just waiting for the new commit

**Pull request merged**